internal/handlers/common/compare.go

```diff
@@ -253,3 +263,30 @@ func compareOrdered[T constraints.Ordered](a, b T) compareResult {
 func compareNumbers(a float64, b int64) compareResult {
 	return compareOrdered(a, float64(b))
 }
+
+// compareDocuments converts two documents to fjson and compares them.
```

Comment:
The comparision of two values should compare them directly, not their serialized forms.
Let's add TODO itdem to improve that in the future.

Reply:
I have processed this case, pls have a look.